.bazelrc

```diff
@@ -1,4 +1,5 @@
 startup --output_base=/tmp/bazel
+startup --host_jvm_args="-Xmx8g"
```

Comment:
This flag would also be used in local builds correct? I am somewhat worried about what impact this might have locally - but I could not see any `OutOfMemoryError`s when building some things like the `session_manager` or the MME core.

Reply:
It is more than the default value (otherwise the test runs on my fork would have failed) - but I was not able to find in the Bazel docs what the default value actually is.

Reply:
I found a comment on [sourcegraph](https://sourcegraph.com/github.com/sorbet/sorbet/-/blob/.bazelrc?L6) that "the default is to use 1/4 of memory":
```
startup --host_jvm_args=-Xmx3072m # we need at least 3G to reliably build, the default is to use 1/4 of memory, which makes flaky on 8G boxes
```

Could a fixed 8GB limit be an issue for devs that have less memory available or can we assume that everyone has at least 8GB to make use of?

Reply:
Discussed offline: using less memory makes the CI jobs fail. We expect dev machines to have >8gb of memory. If we get feed-back for failing CI jobs or dev setup then we can adapt this further.